aten/src/ATen/test/vec_test_all_types.cpp

```diff
@@ -1297,6 +1301,128 @@ namespace {
             },
             test_case);
     }
+    TYPED_TEST(FunctionalBF16Tests, Reduce) {
+      using vec = TypeParam;
+      // Can't use ValueType<TypeParam> here:
+      // Vectorized<BFloat16>::value_type returns uint16_t on AVX2
```

Comment:
Can you please file GH issue for it and reference this piece for removal/update.